pytripgui/field_vc/field_cont.py

```diff
@@ -47,10 +54,33 @@ def set_model_from_view(self):
         else:
             self.model.isocenter = []
 
-        self.model.gantry = self.view.gantry_angle
-        self.model.couch = self.view.couch_angle
+        self.model.display_angles_in_standard = self.view.angles_standard
```

Comment:
I'd rename `self.model.display_angles_in_standard` to `self.model.angles_in_standard` here in all other uses